src/rpc/evo.cpp

```diff
@@ -549,7 +570,293 @@ static UniValue protx_register_wrapper(const JSONRPCRequest& request,
     if (isFundRegister) {
         uint32_t collateralIndex = (uint32_t) -1;
         for (uint32_t i = 0; i < tx.vout.size(); i++) {
-            if (tx.vout[i].nValue == collateralAmount) {
+            if (tx.vout[i].nValue == CDeterministicMN::REGULAR_MASTERNODE_COLLATERAL) {
```

Comment:
don't understand if this is correct, same with the static collateral value above

Reply:
@PastaPastaPasta When calling `protx register_fund` and `register_fund_hpmn` we need to find 1000/4000 Dash in the vout so we can store collateralOutpoint.

Reply:
Well, I think I was more concerned with the duplication of code here more than anything